project/profiles/capacity4more/modules/c4m/user/c4m_user_og/c4m_user_og.module

```diff
@@ -544,43 +553,71 @@ function c4m_user_og_group_actions_cta($group) {
   }
 
   $items = array();
-  // CTAs for the group owner: Manage group and number or new membership
-  // requests.
-  if (c4m_og_user_is_owner('node', $group->nid)) {
+  // CTAs for the group owner and administrators: Manage group and number or
+  // new membership requests.
+  $is_owner = ($user->uid == $group->uid);
+  $group_admins = c4m_og_get_group_admins($group->nid);
+  $is_admin = in_array($user->uid, array_keys($group_admins));
+  if ($is_owner || $is_admin) {
```

Comment:
@DavidHernandez 
can we use `_c4m_features_og_members_is_power_user()` instead of that logic ?